### ISSUE TITLE
Github release trigger fails to publish docker manifest image(s)

### ISSUE THREAD

**Issue by garyschulte** [hyperledger/besu#5521]
### Description
Upon publishing a release, the [release workflow](https://github.com/hyperledger/besu/blob/main/.github/workflows/release.yml)  should publish updated `latest` tags for the corresponding version.  For some reason this task is failing since [23.4.1](https://github.com/hyperledger/besu/actions/runs/5137011125/jobs/9245855066)

This might be dockerhub permissions or problems with self-hosted runners.

### Acceptance Criteria
* triggering the release workflow should work as anticipated

### Steps to Reproduce (Bug)
1. publish a release

**Expected behavior:** [What you expect to happen]
hyperledger/besu:latest tag (among others) get updated to a manifest image of the latest amd64 and arm64 images

**Actual behavior:** [What actually happens]
manifestDockerRelease task fails with an error:
  `refusing to amend an existing manifest list with no --amend flag`

**Frequency:** [What percentage of the time does it occur?]
100%

**Comment by siladu** [comment]
[Last successful run](https://github.com/hyperledger/besu/actions/runs/4884808573) (for 23.4.0) logs look like this:
```
> Task :manifestDockerRelease
Created manifest list docker.io/***/besu:latest-openjdk-17
sha256:95dc6f7295c3cd9ba73a1b87d79f19e1ee451c6079c35e88cb90d1dcea1490fe
Created manifest list docker.io/***/besu:latest-openjdk-17-debug
sha256:f2ad89f52f33f0a98dad34418cfb755ae9072fd32e4654d17df7199bf822296b
Created manifest list docker.io/***/besu:latest-openj9-jdk-17
sha256:07536a4349f8b1e3f297fe101fd71923ff3fd02c38f4036b90621b09b5e018e0
Created manifest list docker.io/***/besu:latest-graalvm
sha256:93f6b652bb[24](https://github.com/hyperledger/besu/actions/runs/4884808573/jobs/8718054965#step:7:25)04ee9[25](https://github.com/hyperledger/besu/actions/runs/4884808573/jobs/8718054965#step:7:26)c2c7dcc0d71ef154d3013a0661d457e59d4dc58c3602e
Created manifest list docker.io/***/besu:latest-openjdk-latest
sha256:6fc0155b547a513de771e5148555091e2c9cc2b7e85f9d0c4ccbfa1961d58e2b
Created manifest list docker.io/***/besu:latest
sha256:95dc6f7[29](https://github.com/hyperledger/besu/actions/runs/4884808573/jobs/8718054965#step:7:30)5c3cd9ba73a1b87d79f19e1ee451c6079c[35](https://github.com/hyperledger/besu/actions/runs/4884808573/jobs/8718054965#step:7:36)e88cb90d1dcea1490fe
```

For 23.4.1, despite the logs, it appears to publish all latest-<variant> tags except the final `latest`.

**Comment by siladu** [comment]
Here you can see the difference between the manually updated `lastest` (@garyschulte ran the gradle task locally) versus all the others that succeeded when using the GHA...

<img width="1265" alt="Screenshot 2023-06-01 at 7 35 04 am" src="https://github.com/hyperledger/besu/assets/2893793/7f7d8864-122f-4b9c-b257-ce2c7c99c120">

**Comment by macfarla** [comment]
@siladu is this one fixed by #5661 ?

**Comment by siladu** [comment]
> @siladu is this one fixed by #5661 ?

Good spot, yes. It hasn't been tested via github yet (only locally) but can always reopen if necessary
